### ISSUE TITLE
Create data preprocessing pipeline

### ISSUE THREAD

**Issue by Manjushwarofficial** [GSSoC-Flow/WardhaMetroFLow#12]
📌 Description: Implement a reusable pipeline for cleaning, transforming, and normalizing metro data.
✅ Task: Use pandas, sklearn.Pipeline, and MinMaxScaler.

**Comment by Parvv04** [comment]
i would like to work on this as well!

**Comment by Manjushwarofficial** [comment]
Sure

**Comment by Parvv04** [comment]
assign me please!

**Comment by Manjushwarofficial** [comment]
Hey @Kritanucoder somebody is already working on this  base on first come first serve basis. I'd suggest to join the project discord from readme and wait for enhancement or create new issues

**Comment by Manjushwarofficial** [comment]
https://discord.gg/QUbJXpNY Hey join this project discord ...somebody is already working on this.
